### PR TITLE
fix default content card blocks

### PR DESCRIPTION
**Pull request by joewhitsitt**
# How to test

```
ddev blt di --site=default && ddev blt ds --site=daly.lab.uiowa.edu
```

```
ddev drush @default.local uli node/2/layout
```

pick a card to inspect. should be minimal plus.

```
ddev drush @labdaly.local uli node/1/layout
```

pick a card to inspect. should be minimal plus.

might have to `blt bis` to get daly.lab to sync.

### CODE REVIEW

File: docroot/profiles/custom/sitenow/sitenow.install [hunk 1]
```diff
@@ -1924,3 +1924,17 @@ function sitenow_update_9052() {
...
+ * Re-run of 9051 to fix recently provisioned sites.
+ */
+function sitenow_update_9053() {
+  _update_all_blocks_by_plugin_id('inline_block:uiowa_card', function (&$component, $block) {
+
+    // Change the format for the card excerpt.
+    if (!is_null($block)) {
+      $block->field_uiowa_card_excerpt->format = 'minimal_plus';
+      $block->save();
+    }
```
Comment (pyrello) [inline docroot/profiles/custom/sitenow/sitenow.install:1940]:
```suggestion
    if (!is_null($block)) {
      $current_format = $block->field_uiowa_card_excerpt->format;
      if (!current_format !== 'minimal_plus') {
        $block->field_uiowa_card_excerpt->format = 'minimal_plus';
        $block->save();
      }
    }
```

**Comment by pyrello** [comment]
Pulling to test.

**Review by pyrello — APPROVED** [review]
Tested. Looks good.

**Comment by joewhitsitt** [comment]
thanks for assisting!

**Pull request merged**